fabric-containers-v0/build.gradle

```diff
@@ -1,5 +1,5 @@
 archivesBaseName = "fabric-containers-v0"
-version = getSubprojectVersion(project, "0.1.2")
+version = getSubprojectVersion(project, "0.2")
```

Comment:
In cases like this the next build would be `0.1.1`, this would be wrong, so I had to bump the minor version.

One way around this would be to create the json file and manually increase the build number, this may be a slight pain to do, but should be possible.